cluster_experiments/power_analysis.py

```diff
@@ -284,8 +284,7 @@ def _non_parallel_loop(
         n_detected_mde = 0
         for _ in tqdm(range(n_simulations), disable=not verbose):
             p_value = self._run_simulation((df, average_effect))
-            if verbose:
-                print(f"p_value of simulation run: {p_value:.3f}")
+            logging.info(f"p_value of simulation run: {p_value:.3f}")
```

Comment:
I'd still add it instead the if verbose, wdyt?

Reply:
Hmm I'd say that using logging you already have a way of controlling the verbosity by setting the log level. In particular, I wanted to take it out from the `if verbose` since in the power_config we haven't parameterized the `verbose` setting and I wanted each simulation p_value to be logged when running simulations on argo (without having to add another parameter to the power_config).

Reply:
but you don't need to add another parameter right? you can call the power_analysis method with verbose=True and you get the printed logs, right?

Reply:
(I agree that you have a way of controlling verbosity, but I think it's friendlier for users to pass the verbose=True or False)

Reply:
Ah true, you're right, it doesn't have to be in the power_config itself!
About the `verbose` argument being "friendlier" than the log level, I don't know, seems a bit subjective^^ I don't have a strong opinion, but if we choose to use the `verbose` argument here then I'd maybe replace the usage of `logging` with a usual `print` statement, as otherwise there could be some confusion if the user sets `verbose=True` but the p_values don't show up. WDYT?

Reply:
yes, agree!

Reply:
then, we revert this, right?

Reply:
okay, done